content/talk_archive.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Past Talks and Seminars
+---
+
+-A quick reference for our past talks by group organizers, members and industry experts
+
+**Beautiful Soup**
+
+Talk on: March 22 
+By: Fernando Corey
+
+[Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/) is a Python library designed for quick turnaround projects like screen-scraping.
+
+[Lecture Slides](https://docs.google.com/presentation/d/14-pxVV1JC-QTYgNGlHeZUcYKG-9maunwL5BkY6Fsaas/edit?usp=sharing)
+
+[Github Repository](https://github.com/fernando-mc/beautiful-soup-web-scraping)
+
+**Flask**
+
+Talk on: March 15
+By: Nick
+
+Google, YouTube, Quora – they are all built in Python.
+
+You can build powerful web applications with Python and have a good time doing so.Learn how to use [Flask framework](http://flask.pocoo.org/) and build a website from scratch. 
+
+[Lecture Slides and Introductory Code](https://github.com/mkpt/flask-api)
+
+-RedmondPython is open to any member who wants to do a quick walkthrough and introduction of your favorite library, framework, python principles and anything pythonic. Please talk to the organizers if you are interested. 
```

Comment:
suggestion: Adding emailto: for organizer or slack channel details to reach out will be helpful here.